editor/export/editor_export_platform.cpp

```diff
@@ -129,8 +129,11 @@ bool EditorExportPlatform::fill_log_messages(RichTextLabel *p_log, Error p_err)
 		}
 		p_log->pop();
 		p_log->add_newline();
+	} else {
+		p_log->add_text(TTR("Check debug output for more details."));
```

Comment:
Wouldn't that also show on a successful build with no warning?

Reply:
I'm not sure if it should be in the `else` branch, it's possible to have warning messages which does not stop export and then fail for another reason.

Maybe we should add it as a static label directly to the dialog UI as a generic reminder, since successfully run can have extra details in the log as well.

Reply:
I think we should _not_ show the text if there is already helpful messages in the debug log.

When I see the text "Check debug output for more details.", I am pressured to check the debug log, which could mean clicking X/Back on 2 different windows, which I will need to traverse again when I find out the debug log contains no new/helpful information.

I think this window should be a elegant replacement of the other debug output, and we should make sure all errors are correctly reported in it.